### DISCUSSION TITLE
Change language

### DISCUSSION THREAD

**Discussion by JeremyMeissner** [javalent/obsidian-leaflet#283] [General]
Hello, is there any way I can set the language of the entire map to English?

**Comment by valentine195** [comment]
You can change the tile server used using the `tileServer` parameter, and turn off the regular OpenStreetMap tile server using `osmLayer: false`.

If you can find a tile server with all English names, please post it here!

**Comment by JeremyMeissner** [comment]
Thank you! I found two basic ones that work

English map **Lightmode**: 
```
osmLayer: false
tileServer: https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png
``` 

English map **Darkmode**: 
```
osmLayer: false
tileServer: https://{s}.basemaps.cartocdn.com/dark_all/{z}/{x}/{y}{r}.png
```
